backend-rust/src/migrations.rs

```diff
@@ -0,0 +1,264 @@
+use anyhow::Context;
```

Comment:
Instead of having schema versions, why dont we just ensure that the sql is the same as what is defined within the .sql files?

Reply:
Not sure, I understand what you are suggesting.
If you mean ask the database for the schema and then compare with the sql files, it seems very complex solution when we start altering tables and remove stuff.

Reply:
Basically this relates quite heavily to the other conversation so I am closing this.